### ISSUE TITLE
Publish Hardware Report data

### ISSUE THREAD

**Issue by openjck** [mozilla/Fx_Usage_Report#29]
We are currently transforming the [current Hardware Report JSON](https://analysis-output.telemetry.mozilla.org/game-hardware-survey/data/hwsurvey-weekly.json) into the format that the Public Data Report website expects. If *FX_Usage_Report* could output this data directly, in the same format as the *Health* and *Usage* datasets, the transformation would be unnecessary and the performance of the site would be better.

It would be nice if this could be done before release, but I don't think it's a blocker. The transformer we wrote does work, even if it is a little slow.

In addition to changing the format of the JSON, our transformer also changes the organization of the data. It removes some populations, renames some populations, and combines some populations into bigger ones. These changes are documented in the file [population-modifications.json](https://github.com/mozilla/fhwr-unflattener/blob/master/src/population-modifications.json). Ideally these changes would be made on your end before spitting the data out.

**tl;dr:** Please publish the hardware data directly so that the extra transformation step becomes unnecessary. The resulting JSON should be identical to [what our transformer spits out](http://fhwr-unflattener.herokuapp.com/).

**Comment by openjck** [comment]
We're going to use [workshop](https://github.com/mozilla/workshop) to achieve this, rewriting it as a Lambda task that writes to S3 in the process.
